actors/test/committed_capacity_scenario_test.go

```diff
@@ -239,9 +270,96 @@ func TestReplaceCommittedCapacitySectorWithDealLadenSector(t *testing.T) {
+	// modified if this no longer holds.
+	oldDlIdx, _ := vm.SectorDeadline(t, v, minerAddrs.IDAddress, sectorNumber)
+	newDlIdx, _ := vm.SectorDeadline(t, v, minerAddrs.IDAddress, ccSectorNumber)
+	require.Equal(t, oldDlIdx, newDlIdx)
+
+	t.Run("miner misses first PoSt of replacement sector", func(t *testing.T) {
+		// advance to proving period end of new sector
+		dlInfo, _, tv := vm.AdvanceTillProvingDeadline(t, v, minerAddrs.IDAddress, sectorNumber)
+		tv, err = tv.WithEpoch(dlInfo.Last())
+		require.NoError(t, err)
+
+		// run cron to penalize missing PoSt
+		_, code = tv.ApplyMessage(builtin.SystemActorAddr, builtin.CronActorAddr, big.Zero(), builtin.MethodsCron.EpochTick, nil)
+		require.Equal(t, exitcode.Ok, code)
+
+		vm.ExpectInvocation{
+			To:     builtin.CronActorAddr,
+			Method: builtin.MethodsCron.EpochTick,
+			SubInvocations: []vm.ExpectInvocation{
+				{To: builtin.StoragePowerActorAddr, Method: builtin.MethodsPower.OnEpochTickEnd, SubInvocations: []vm.ExpectInvocation{
+					{To: minerAddrs.IDAddress, Method: builtin.MethodsMiner.OnDeferredCronEvent, SubInvocations: []vm.ExpectInvocation{
+						{To: builtin.RewardActorAddr, Method: builtin.MethodsReward.ThisEpochReward},
+						{To: builtin.StoragePowerActorAddr, Method: builtin.MethodsPower.CurrentTotalPower},
+						// power is removed for old sector and pledge is burnt
+						{To: builtin.StoragePowerActorAddr, Method: builtin.MethodsPower.UpdateClaimedPower},
+						{To: builtin.BurntFundsActorAddr, Method: builtin.MethodSend},
+						{To: builtin.StoragePowerActorAddr, Method: builtin.MethodsPower.UpdatePledgeTotal},
+						{To: builtin.StoragePowerActorAddr, Method: builtin.MethodsPower.EnrollCronEvent},
+					}},
+					{To: builtin.RewardActorAddr, Method: builtin.MethodsReward.UpdateNetworkKPI},
+				}},
+				{To: builtin.StorageMarketActorAddr, Method: builtin.MethodsMarket.CronTick, SubInvocations: []vm.ExpectInvocation{}},
+			},
+		}.Matches(t, tv.LastInvocation())
+
+		// miner's power is removed for old sector because it faulted and not added for the new sector.
+		minerPower = vm.MinerPower(t, tv, minerAddrs.IDAddress)
+		networkStats = vm.GetNetworkStats(t, tv)
+		assert.Equal(t, big.Zero(), minerPower.Raw)
+		assert.Equal(t, big.Zero(), minerPower.Raw)
+		assert.Equal(t, big.Zero(), networkStats.TotalBytesCommitted)
+		assert.Equal(t, big.Zero(), networkStats.TotalQABytesCommitted)
+	})
+
 	// advance to proving period and submit post
 	dlInfo, pIdx, v = vm.AdvanceTillProvingDeadline(t, v, minerAddrs.IDAddress, ccSectorNumber)
 
+	t.Run("miner skips replacing sector in first PoSt", func(t *testing.T) {
+		tv, err := v.WithEpoch(v.GetEpoch()) // create vm copy
+		require.NoError(t, err)
+
+		submitParams = miner.SubmitWindowedPoStParams{
+			Deadline: dlInfo.Index,
+			Partitions: []miner.PoStPartition{{
+				Index: pIdx,
+				// skip cc upgrade
+				Skipped: bitfield.NewFromSet([]uint64{uint64(ccSectorNumber)}),
```

Comment:
The name`ccSectorNumber` is quite confusing here. A committed capacity sector is a sector that has no deals. This, however, is the a sector that has deals that is replacing a committed capacity sector.

Please rename `sectorNumber` and `ccSectorNumber`. Perhaps:
- ccSectorNumber, upgrade/replacementSectorNumber?
- oldSectorNumber, newSectorNumber?
Use those terms consistently in the comments and other code (you already use old/new a lot).

Name `sectorPower` to match.